src/components/progress-indicator/progress-indicator.js

```diff
@@ -0,0 +1,187 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+import clsx from 'clsx'
+
+import { makeStyles } from '../../utils/make-styles'
+import { CheckBold } from '../../icons'
+import Spin from './spin'
```

Comment:
how about adding it into icon?

Reply:
There's a lot of unique styles for this icon that don't make it useable outside this context, it has to be white for example (different layers of opacity).